app/cinema/hall.py

```diff
@@ -0,0 +1,16 @@
+from app.people.cinema_staff import Cleaner
+
+
+class CinemaHall:
+    def __init__(self, number: int) -> None:
+        self.number = number
+
+    def movie_session(self,
+                      movie_name: str,
+                      customers: list,
+                      cleaning_staff: "Cleaner") -> None:
+        print(f'\"{movie_name}\" started in hall number {self.number}.')
```

Comment:
```suggestion
        print(f'"{movie_name}" started in hall number {self.number}.')
```
you have string with `'`, so you can use just `"` inside